opcua/common/ua_utils.py

```diff
@@ -238,3 +238,36 @@ def get_variable_basetype(server, datatype_id):
     if not dtype_supers_nodeids:
         raise ua.UaError("Datatype must be a subtype of builtin types %s" % datatype_id.nodeid)
     return dtype_supers_nodeids[0]
+
+def get_nodes_of_namespace(server, namespaces=[]):
+    """
+    Get the nodes of one or more namespaces .      
+    Args:
+        server: opc ua server to use
+        namespaces: list of string uri or int indexes of the namespace to export
+    Returns:
+        List of nodes that are part of the provided namespaces
+    """
+    ns_available = server.get_namespace_array()
+    ns_count = len(ns_available)
+
+    if not namespaces:
+        namespaces = ns_available[1:]
+
+    if type(namespaces) is not list and type(namespaces) in [str, int]:
```

Comment:
for some reasons it is advices to use isintance(a, (t1, t2)) instead of type ==. Not very important here. but good habit